### PR TITLE
Examples: Prevent focus into closed offcanvas

### LINKED ISSUE #25441: Off-canvas example: content still focusable when off-canvas menu collapsed
On small screen sizes, when the top navigation turns to off-canvas, the links/controls inside it are still keyboard-focusable despite not being visible

https://getbootstrap.com/docs/4.0/examples/offcanvas/

Normally we'd add `display:none` when the menu is closed, but flipping between `display:none` and `display:block` here kills the animation.

If `inert` attribute was well supported cross-browser, that would be an approach...but it's a bit early for that (perhaps using polyfill?).

But in short: currently this is a classic example of what not to do, accessibility-wise.

### PR DESCRIPTION
**Pull request by ysds**
Fixes #25441

### CODE REVIEW

File: docs/4.0/examples/offcanvas/offcanvas.css [hunk 1]
```diff
@@ -16,14 +16,15 @@ body {
...
     overflow-y: auto;
+    visibility: hidden;
     background-color: var(--gray-dark);
-    transition: -webkit-transform .3s ease-in-out;
-    transition: transform .3s ease-in-out;
-    transition: transform .3s ease-in-out, -webkit-transform .3s ease-in-out;
-    -webkit-transform: translateX(100%);
+    transition: visibility .3s ease-in-out, -webkit-transform .3s ease-in-out;
+    transition: visibility .3s ease-in-out, transform .3s ease-in-out;
+    transition: visibility .3s ease-in-out, transform .3s ease-in-out, -webkit-transform .3s ease-in-out;    -webkit-transform: translateX(100%);
```
Comment (patrickhlauke) [inline docs/4.0/examples/offcanvas/offcanvas.css]: needs a line break here after `;` to pass linting

**Review by patrickhlauke — CHANGES_REQUESTED** [review]

File: docs/4.0/examples/offcanvas/offcanvas.css (hunk 1 above)
Comment (patrickhlauke) [reply docs/4.0/examples/offcanvas/offcanvas.css]: apart from that looking good, and i learned something new here (thought visibility couldn't be transitioned)

Comment (ysds) [reply docs/4.0/examples/offcanvas/offcanvas.css]: Oh, sorry. I fixed it.

**Review by patrickhlauke — APPROVED** [review]

**Comment by patrickhlauke** [comment]
Perfect, thank you.

**Pull request merged**